challenges.js

```diff
@@ -18,16 +18,19 @@ function sayHello() {
 }
 
 
-/*-----------------------------------------------------------------
-Challenge: 01-addOne
+-----------------------------------------------------------------
+// Challenge: 01-addOne
 
-Difficulty: Basic
+// Difficulty: Basic
 
-Prompt:
+// Prompt:
 
-Write a function called addOne that takes a single number as an argument and returns that number plus 1.
+// Write a function called addOne that takes a single number as an argument and returns that number plus 1.
 
-Examples:
+// // Examples:
+function addOne(num) {
+  return num + 1
+}
```

Comment:
Well done! This is great! Any other feedback etc..

Reply:
Thanks!